examples/Recipies/initialization/__main__.py

```diff
@@ -0,0 +1,32 @@
+"""Example to demonstrate how to implement configurable module globals and class variables
+
+Run this example with the following command:
+python ./__main__.py -c ./the_config.toml -w ./the_config.toml
```

Comment:
So a `__main__` is usually an entry point for a module, i've not really seen it called like this.

Can we not actually do `python . -c ...`, or `python initialization` with the last one from the Recipes folder?

Otherwise I think it makes more sense to just name the file `main.py`

Reply:
https://docs.python.org/3/library/__main__.html#:~:text=__main__%20is%20the,entry%20point%20to%20the%20application.
and
https://loguru.readthedocs.io/en/stable/resources/recipes.html#configuring-loguru-to-be-used-by-a-library-or-an-application
Hence, __main__.py is usually the entry point of an application and an application is what I want to target in the example.

I will make a subfolder my_application to make this more explicit. Another example will come for my_library